api_docs/writing-bots.md

```diff
@@ -17,24 +17,28 @@ On this page you'll find:
 
 ## Installing a development version of the Zulip bots package
 
+{start_tabs}
```

Comment:
Another spot where we could clean up the instructions. They are the reverse of what we normally do with the instruction text after the code / command line example.

Reply:
How about:

![image](https://github.com/zulip/zulip/assets/2343554/ee4c2bf2-5fb1-41cc-a871-b64a9d9064cb)